### ISSUE TITLE
ENH [command name]: [feature request title]

### ISSUE THREAD

**Issue by Lestropie** [MRtrix3/mrtrix3#3038]
https://github.com/thalamicseg/hipsthomasdocker

The dependency is way too large to be integrated into the MRtrix container, and I'd prefer to not have any MRtrix scripts that involve executing Docker containers, given that MRtrix commands may themselves be invoked from inside a Docker container. So I think the right approach is for `5ttgen hsvs` to take as an optional input a path to the file / directory created by that tool and perform the requisite conversion / import.

- [ ] Additional choice to existing `-thalami` option
- [ ] If this option is selected, new command-line option must be specified that points to the derivatives of that tool
    (If `-thalami` option is omitted, but this new option is included, infer the corresponding thalamic segmentation mode)
- [ ] Code branch to import data from this location if specified, bypassing execution of other commands relating to thalamic segmentation from other sources
- [ ] If tool produces only segmentations of subnuclei, combine these into a single thalamic parcel
- [ ] If only a T1w-resolution binary mask is produced, perform `voxel2mesh | meshsmooth | mesh2voxel` motif to get partial volume image
- [ ] Ensure that thalamic partial volume images from this source are integrated into downstream computations to produce the 5TT image

**Comment by anagrammarian** [comment]
Yes, the idea is to let people use the outputs of HIPS-THOMAS which they should install and run independently and most would have. Iin another week or so, it will be the entire subcortex + thalamic nuclei. Since we synthesize WMn contrast images, we are sensitive to boundaries etc and generally more accurate TNS.
